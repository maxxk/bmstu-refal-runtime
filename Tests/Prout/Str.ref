$ENTRY Go
{
	= <Prout 'String1' ' text between ' 'String2' >
}
