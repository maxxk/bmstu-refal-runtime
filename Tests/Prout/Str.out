String1 text between String2
